Tweak result caching quickstart (#153)
spiceai/quickstarts: caching/README.md

spice run
```

**Step 2.** Add the TPC-H Benchmark Spicepod

> In separate terminal, using the same directory, add the TPC-H Benchmark Spicepod to the project.

Author: ```suggestion
In a separate terminal, using the same directory, add the TPC-H Benchmark Spicepod to the project.
```

> **Step 2.** Add the TPC-H Benchmark pod

Reviewer: ```suggestion
**Step 2.** Add the TPC-H Benchmark Spicepod
```

> In another terminal window in the same directory, add the TPC-H Benchmark pod to the Spice runtime.

Reviewer: ```suggestion
In separate terminal, using the same directory, add the TPC-H Benchmark Spicepod to the project.
```

2024-08-05T05:26:52.871013Z  INFO runtime: Dataset nation registered (s3://spiceai-demo-datasets/tpch/nation/), results cache enabled.
2024-08-05T05:26:54.201229Z  INFO runtime: Dataset orders registered (s3://spiceai-demo-datasets/tpch/orders/), results cache enabled.
2024-08-05T05:26:55.583954Z  INFO runtime: Dataset part registered (s3://spiceai-demo-datasets/tpch/part/), results cache enabled.
2024-08-05T05:26:56.933827Z  INFO runtime: Dataset partsupp registered (s3://spiceai-demo-datasets/tpch/partsupp/), results cache enabled.
2024-08-05T05:26:58.182547Z  INFO runtime: Dataset region registered (s3://spiceai-demo-datasets/tpch/region/), results cache enabled.
2024-08-05T05:26:59.501475Z  INFO runtime: Dataset supplier registered (s3://spiceai-demo-datasets/tpch/supplier/), results cache enabled.
```

Notice the following line confirming the default cache configuration with cached items expiration time of 1 second is loaded.

```bash
2024-08-05T05:25:10.628875Z  INFO runtime: Initialized results cache; max size: 128.00 MiB, item ttl: 1s
```

**Step 3.** Increase the cached items' expiration time

> Stop Spice with `Ctrl-C` and use a text editor to open the `spicepod.yaml` file. Add a custom in-memory caching configuration below to increase the cached items' duration to `5 minutes`. Read [Spice.ai OSS Docs: Results Caching](https://docs.spiceai.org/features/caching) to learn more about the available configuration parameters.

Reviewer: ```suggestion
Using `Ctrl-C`, stop Spice and use a text editor to open the `spicepod.yaml` file. Add a custom in-memory caching configuration below to increase the cached items' duration to `5 minutes`. Read [Spice.ai OSS Docs: Results Caching](https://docs.spiceai.org/features/caching) to learn more about the available configuration parameters.
```
